tests/tests.rs

```diff
@@ -7,7 +7,7 @@ use futures::{
     stream::{futures_unordered::FuturesUnordered, FuturesOrdered, StreamExt, TryStreamExt},
     TryFutureExt,
 };
-use pearl::{BloomProvider, Builder, Meta, Storage};
+use pearl::{BloomProvider, Builder, Meta, ReadResult, Storage};
```

Comment:
We should have a test for `read_all` when deleted records presented in the middle